dotCMS/src/main/java/com/dotmarketing/util/ImportExportUtil.java

```diff
@@ -1428,7 +1428,6 @@ private void doXMLFileImport(File f, PrintWriter out, ObjectFilter filter)throws
                         }
                         else if(obj instanceof MultiTree){
                             final MultiTree t = (MultiTree) obj;
-                            t.setPersonalization(MultiTree.DOT_PERSONALIZATION_DEFAULT);
```

Comment:
Again, this will overwrite any personalization that exists in the starter, unless I am missing something?